wDeveloper/OpenSourceProjects.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Open Source Projects
+filename: "OpenSourceProjects.md"
+layout: custom
+keywords: []
+headings: ["Overview", "BeetleETL", "Data-Latinizer", "ScriptMySQLServer", "Powershell DB Tools GUI"]
+links: ["#beetleetl", "#data-latinizer", "#scriptmysqlserver", "#powershell-db-tools-gui", "https://pypi.org/project/BeetleETL/", "https://gitlab.com/Open-Interject/Beetle-ETL", "https://gitlab.com/Open-Interject/Data-Latinizer", "https://gitlab.com/Open-Interject/ScriptMySQLServer", "https://gitlab.com/Open-Interject/PowershellDBToolsGui"]
+image_dir: ""
+images: []
+description: Interject hosts a variety of open-source projects aimed at providing useful tools for data handling, automation, and database management.
+---
+* * *
+
+## Overview
+
+Interject hosts a variety of open-source projects aimed at providing useful tools for data handling, automation, and database management.
```

Comment:
we should add a comment here that lets people know all these projects are not actively maintained and they should use at their own risk understanding this.